public/app/features/alerting/AlertTab.tsx

```diff
@@ -205,19 +228,36 @@ class UnConnectedAlertTab extends PureComponent<Props, State> {
     };
 
     return (
-      <EditorTabBody toolbarItems={toolbarItems}>
-        <div aria-label={selectors.components.AlertTab.content}>
-          {alert && hasTransformations && (
-            <Alert
-              severity={AppNotificationSeverity.Error}
-              title="Transformations are not supported in alert queries"
-            />
-          )}
-
-          <div ref={element => (this.element = element)} />
-          {!alert && !validatonMessage && <EmptyListCTA {...model} />}
-        </div>
-      </EditorTabBody>
+      <>
+        <CustomScrollbar autoHeightMin="100%">
+          <Container padding="md">
+            <div aria-label={selectors.components.AlertTab.content}>
+              {alert && hasTransformations && (
+                <Alert
+                  severity={AppNotificationSeverity.Error}
+                  title="Transformations are not supported in alert queries"
+                />
+              )}
+
+              <div ref={element => (this.element = element)} />
+              {alert && (
+                <HorizontalGroup>
+                  <Button onClick={() => this.onToggleModal('showStateHistory')}>State history</Button>
+                  <Button onClick={() => this.onToggleModal('showTestRule')}>Test rule</Button>
```

Comment:
Think we can use secondary buttons here to be in line with other tabs:
```suggestion
                  <Button onClick={() => this.onToggleModal('showStateHistory')} variant="secondary">State history</Button>
                  <Button onClick={() => this.onToggleModal('showTestRule')} variant="secondary">Test rule</Button>
```